index.html

```diff
@@ -3178,30 +3180,33 @@ <h3>Issuer</h3>
 available to the <a>verifier</a>. For example, an <a>issuer</a> can publish
 information containing the public keys it uses to digitally sign
 <a>verifiable credentials</a> that it issued. This metadata is relevant
-when checking the proofs on the <a>verifiable credential</a>.
+when checking the proofs on the <a>verifiable credentials</a>.
         </p>
       </section>
 
       <section>
         <h3>Subject</h3>
 
         <p>
-The value associated with the <code>id</code> <a>property</a> for each
-<code>credentialSubject</code> MUST identify a <a>subject</a> to the
-<a>verifier</a>. For example, if a <a>subject</a> is identified and the
-<a>verifier</a> has public key metadata used for authentication purposes that
-is related to the <a>subject</a>, the <a>verifier</a> MAY be able to
-authenticate the <a>subject</a> using a signature generated by the
-<a>subject</a> contained in the <a>verifiable presentation</a>.
+In the <a>verifiable credentials</a> <a>presented</a> by a <a>holder</a> on
+behalf of a <a>subject</a>, the value associated with the <code>id</code>
```

Comment:
This is wrong on two counts:
a) a VC can contain multiple credentialSubjects, not just one
b) the VCs may not be presented on behalf of the subject, but may be presented on behalf of the holder him/herself

Reply:
Removed "on behalf of the subject".
I believe that the word "each" in "...the value associated with the id property for each credentialSubject MUST identify a subject..." covers the case where a VC contains multiple credentialSubjects.